src/go/gcsrunner/start_envoy.go

```diff
@@ -51,6 +51,7 @@ func StartEnvoyAndWait(signalChan chan os.Signal, opts StartEnvoyOptions) error
 		"--log-path", opts.LogPath,
 		"--log-format", "%L%m%d %T.%e %t envoy] [%t][%n]%v",
 		"--log-format-escaped",
+		"--allow-unknown-static-fields",
```

Comment:
It is kind of dangerous to turn this on.  Default is to make sure envoy config can be understood by the envoy.  

I prefer not to enable this

Reply:
I guess, our config is always generated by config mgr,  it should not have typo.  
It is ok to turn this on then